Recipes/CGC/files_copyFromMyProject.ipynb

```diff
@@ -107,7 +107,7 @@
     "\n",
     "# LIST all files in the source and target project\n",
     "my_files = api.files.query(limit = 100, project = my_project.id).all()\n",
-    "source_files = api.files.query(limit = 100, project = source_project.id).all()\n",
+    "source_files = list(api.files.query(limit = 100, project = source_project.id).all())\n",
```

Comment:
Generators work as list when used only once in the code. Once you do this:
source_files = api.files.query(limit = 100, project = source_project.id).all()
source_file_names = [f.name for f in source_files]

source_files is empty (as it has been used once). As you do not have source_file initialized yet, the script had a bug. To get around that, I converted to list and did this:
source_file = [curr_file for curr_file in source_files if curr_file.name == source_file_names[f_index]][0]

To do that, you need to either recreate source_file_names with a new API call or make it a list earlier.  Hence, the suggested change. But [@SenadI](https://github.com/SenadI) can tell me what he recommends in this case.